desc/compute/_metric.py

```diff
@@ -1823,3 +1823,74 @@ def _gradtheta(params, transforms, profiles, data, **kwargs):
 def _gradzeta(params, transforms, profiles, data, **kwargs):
     data["|grad(zeta)|"] = jnp.sqrt(data["g^zz"])
     return data
+
+
+@register_compute_fun(
+    name="gbdrift",
+    label="\\mathrm{gradB-drift}",
+    units="~",
+    units_long="None",
+    description="Binormal component of th geometric part of the gradB drift"
```

Comment:
typo, th -> the. Also probably don't need all the line breaks

Reply:
Corrected.

Reply:
Do need line breaks to avoid 88 character limit.